### PR TITLE
fix: coordinates bug on pdf parsing

### PR DESCRIPTION
**Pull request by amanda103**
Addresses: https://github.com/Unstructured-IO/unstructured/issues/1460

We were raising an error with invalid coordinates, which prevented us from continuing to return the element and continue parsing the pdf. Now instead of raising the error we'll return early.

to test:
```
from unstructured.partition.auto import partition

elements = partition(url='https://www.apple.com/environment/pdf/Apple_Environmental_Progress_Report_2022.pdf', strategy="fast")
```

### CODE REVIEW

File: unstructured/partition/pdf.py [hunk 1]
```diff
@@ -761,7 +761,7 @@ def check_coords_within_boundary(
         a float ranges from [0,1] to scale the horizontal (x-axis) boundary
     """
     if not coord_has_valid_points(coordinates) and not coord_has_valid_points(boundary):
-        raise ValueError("Invalid coordinates.")
+        return False
```
Comment (cragwolfe) [inline unstructured/partition/pdf.py:764]: worth adding a trace detail message i think, like: https://github.com/Unstructured-IO/unstructured/blob/9a3e24fcbb4efef30d5f046952e794666345b7ea/unstructured/partition/utils/sorting.py#L70
.
is there a single page this fails on that could be added to a test? one could extract the problematic page with something like:

    qpdf --empty --pages orignal.pdf 89 -- ~/tmp/docs/single-page-p89.pdf

finally, does not need to be addressed in this PR, but what is the why for the invalid coordinate? is it negative values? (which i think are actually legit in PDF's in certain cases)

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,8 @@
 
 ### Fixes
 
+* **GH Issue 1460** Fixes a bug that prevented certain pdf's from bypassing new coordinate logic
```
Comment (cragwolfe) [inline CHANGELOG.md:12]:
```suggestion
* ***Fixes an issue that caused a partition error for some PDF's.** Fixes GH Issue 1460 by bypassing a coordinate check if an element has invalid coordinates.
```

File: unstructured/partition/pdf.py (hunk 1 above)
Comment (amanda103) [reply unstructured/partition/pdf.py:764]: Sounds good - I'll add trace detail and a test!

And yes - the values that it was erroring on were negative.

Comment (amanda103) [reply unstructured/partition/pdf.py:764]: here's where it was failing: https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/partition/utils/sorting.py#L30

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,7 @@
 
 ### Fixes
 
+* ***Fixes an issue that caused a partition error for some PDF's.** Fixes GH Issue 1460 by bypassing a coordinate check if an element has invalid coordinates.
```
Comment (cragwolfe) [inline CHANGELOG.md:13]:
```suggestion
* ***Fixes an issue that caused a partition error for some PDF's.** Fixes GH Issue 1460 by bypassing a coordinate check if an element has invalid coordinates.

```
newline

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-## 0.10.16-dev2
+## 0.10.16
```
Comment (cragwolfe) [inline CHANGELOG.md:2]:
```suggestion
```
rm line

**Review by cragwolfe — APPROVED** [review]
🎉

**Pull request merged**